src/resolvers/user.ts

```diff
@@ -94,71 +96,77 @@ const resolver: Resolvers = {
       const auth = await getUser();
       return auth;
     },
-    users: async (_, args, { verifyUser, models }): Promise<User[]> => {
+    users: async (
+      _,
+      args,
+      { verifyUser, models },
+    ): Promise<UsersConnection> => {
       const { User: userModel } = models;
       const auth = verifyUser();
       checkAuth(auth);
 
-      const { user, includeUser, filter, first, after } = args;
+      const { filter, user, includeUser, pageSize = 20, after } = args;
+      let qryUser: object = {};
+      let qryAfter: object = {};
+      let qryIncludeUser: object = {};
 
```

Comment:
Same issue as above. It looks like we need discussion or comes up with unopinionated style of code.